internal/spec/protection.go

```diff
@@ -119,6 +125,7 @@ func (bp *Protection) ToGithub(pr *github.ProtectionRequest) error {
 	if bp.Permissions.Restrict {
 		pr.Restrictions = bp.permissionsToGithub()
 	}
+	pr.RequiredConversationResolution = &bp.ConversationResolution
```

Comment:
It seems we should set it to `pr.RequiredConversationResolution.Enabled`, not to `pr.RequiredConversationResolution`

Reply:
No, `pr` is the `github.ProtectionRequest` struct, its `RequiredConversationResolution` field type is `*bool`.